apps/ember-test-app/app/components/f/mktg/footer.gts

```diff
@@ -0,0 +1,112 @@
+  hasDivider = false;
+
+  @action
+  update(key: string, value: unknown) {
+    this[key] = value;
+  }
+
+  <template>
+    <FreestyleSection @name="Footer" as |Section|>
+      <Section.subsection @name="Basics">
+        <FreestyleUsage>
+          <:example>
+            <Footer class={{this.class}} @hasDivider={{this.hasDivider}}>
+              <:nav>
+                <Button class="ms-2 rounded-pill btn-light">Nav 1</Button>
+                <a
+                  href=""
+                  class="text-light link-underline link-underline-opacity-0"
+                >
+                  Nav 2
+                </a>
+                <a
+                  href=""
+                  class="text-light link-underline link-underline-opacity-0"
+                >
+                  Nav 3
+                </a>
+                <a
+                  href=""
+                  class="text-light link-underline link-underline-opacity-0"
+                >
+                  Nav 4
+                </a>
+                <a
+                  href=""
+                  class="text-light link-underline link-underline-opacity-0"
+                >
+                  Nav 5
+                </a>
+              </:nav>
+              <:social-media>
+                <a href=""><i class="text-light bi-facebook h4" /></a>
+                <a href=""><i class="text-light bi-twitter h4" /></a>
+                <a href=""><i class="text-light bi-instagram h4" /></a>
+                <a href=""><i class="text-light bi-linkedin h4" /></a>
+                <a href=""><i class="text-light bi-youtube h4" /></a>
+              </:social-media>
+              <:brand>
+                <div>
+                  <img
+                    src="https://imageplaceholder.net/50x50"
+                    alt="Sample icon"
+                  />
+                </div>
+                <p href="" class="text-light mb-0">Brand</p>
+              </:brand>
+              <:legal>
+                <a href="" class="text-light">Legal 1</a>
+                <a href="" class="text-light">Legal 2</a>
+              </:legal>
+            </Footer>
+          </:example>
+          <:api as |Args|>
+            <Args.String
+              @description="The class to apply to the footer. Note that this is not an argument but rather a class applied directly to the footer"
+              @name="class"
+              @value={{this.class}}
+              @onInput={{fn this.update "class"}}
+            />
+            <Args.Bool
+              @description="When true, the footer will render a divider between the top and bottom sections. Note that the default value is false"
+              @name="divider"
+              value={{this.hasDivider}}
+              @onInput={{fn this.update "hasDivider"}}
+              @defaultValue={{false}}
+            />
+            <Args.Yield
+              @description="Named yield block that renders navigational content in the top left of the footer"
```

Comment:
I think all of the descriptions can be condensed:
```suggestion
              @description="Renders navigational content in the top left"
```
1. The type `Yield` implies that this is a yield block, and providing a name implies it has a name.
2. We're in the context of the footer, so I don't think we need to specify where these blocks go